development/src/pages/home/home.scss

```diff
@@ -15,7 +15,7 @@ ul, ol{
 	position: relative;
 }
 
-.profile-section {
+.profile {
 	display: flex;
```

Comment:
@Adititomar123 please fix the indentation

Reply:
fixed